src/login/pam_systemd.c

```diff
@@ -28,6 +28,7 @@
 #include "path-util.h"
 #include "process-util.h"
 #include "socket-util.h"
+#include "string-util.h"
```

Comment:
I guess `#include "stdio-util.h" should be here as well to fix the CentOS CI error

```
../src/login/pam_systemd.c: In function ‘pam_sm_open_session’:
../src/login/pam_systemd.c:433:17: error: implicit declaration of function ‘xsprintf’ [-Werror=implicit-function-declaration]
                 xsprintf(rt, "/run/user/"UID_FMT, pw->pw_uid);
```